client/components/sidebar/products/project-sidebar.service.js

```diff
@@ -1,10 +1,18 @@
 class CloudProjectSidebar {
-  constructor($translate, SidebarMenu, TARGET) {
+  constructor($translate, OvhApiMe, SidebarMenu, URLS, TARGET) {
     this.$translate = $translate;
+    this.OvhApiMe = OvhApiMe;
     this.SidebarMenu = SidebarMenu;
+    this.URLS = URLS;
     this.TARGET = TARGET;
 
     this.type = 'PROJECT';
+
+    this.locale = null;
+    this.OvhApiMe.v6().get().$promise
+      .then((user) => {
+        this.locale = user.ovhSubsidiary;
```

Comment:
Actually, `locale` was not really needed, `TARGET`/`coreConfig.getRegion()` was enough 😉